src/Chainweb/Chainweb.hs

```diff
@@ -401,6 +432,7 @@ withChainwebInternal conf logger peer rocksDb inner = do
     v = _configChainwebVersion conf
     cids = chainIds v
     cwnid = _configNodeId conf
+    enableTxsReintro =  _enableConfigEnabled $ _configReintroTxs conf
```

Comment:
This could be moved into a where clause in the `go` function instead of passing it as an argument to `go`. However, I am not sure what is better performance wise. It depends on tail-recursion-optimization and common-subexpression-elimination play together in GHC. In the end, I may not make a difference at all.

Reply:
I don't know if there are performance differences, but in general this situation or the "let vs where" question - I generally treat it like I was writing a proof -- is the main function more understandable if you define the sub-item first? or is it just noise that is better to fill in afterward?